content/docs/user-guide/visualizing-plots.md

```diff
@@ -276,6 +276,8 @@ The `linear` template is the default. It can be changed with the `--template`
 (`-t`) option of `dvc plots show` and `dvc plots diff`. The argument provided to
 `--template` can be a (built-in) template name or a path to a [custom template].
 
+[custom templates]: /doc/command-reference/plots/templates
```

Comment:
> Yes we should consolidate...

Like this

```suggestion
`--template` can be a (built-in) template name or a path to a
[custom template][plots-templates].

[plots-templates]: /doc/command-reference/plots/templates
```

Reply:
And reuse in line 307 too, ideally.

Reply:
p.s. it's also a bit strange that the ref is in line 299, interrupting a bullet list.